### PR TITLE
[Pagination] Remove NDL fallbacks

### PR DESCRIPTION
**Pull request by aveline**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/polaris-ux/issues/554

### WHAT is this pull request doing?

Removes fallback references to `newDesignLanguage` in the Pagination component

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
- [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit

### CODE REVIEW

**Comment by aveline** [comment]
I am wondering whether the `interface PaginationProps` can be removed altogether?

**Review by kyledurand — APPROVED** [review]
Just pushed up a couple of small changes. I was going to try to explain how to remove the extend from the Pagination props but would have been too much writing and confusing I think 😄

File: src/components/Pagination/Pagination.scss [hunk 1]
```diff
@@ -1,187 +0,0 @@
-@import '../../styles/common';
```
Comment (alex-page) [inline src/components/Pagination/Pagination.scss:1]: This is so awesome we can remove this entire file 🙌 🏄‍♂️ 🌌

**Review by alex-page — APPROVED** [review]
Amazing work @aveline 🔥 🔥 🔥

**Pull request merged**